customer-bill-management/src/main/java/org/fiware/tmforum/customerbillmanagement/configuration/ApiExtensionProperties.java

```diff
@@ -0,0 +1,15 @@
+package org.fiware.tmforum.customerbillmanagement.configuration;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+import lombok.Data;
+
+@ConfigurationProperties("apiExtension")
+@Data
+public class ApiExtensionProperties {
```

Comment:
The controller currently does not go for the existence of this bean but for the property directly

Reply:
Yes. It should. If not, it would also enable the controller in case of apiExtension.enabled=false

Reply:
Sorry had a wrong thought: You are right, it doesn't make sense to go for the existence of this bean as its always there. But what do we need this property for anyway? As far as I can see you dont use in the code and in the test code you generate the mocks but control the controller creation via the property directly. Or am I overlooking something?

Reply:
Because its not part of the standard api, thus should be only explicitly be enabled. The class is mainly for documentation and potentially extension in the future. Since all properties are build through such bean, I wanted to keep it the same way.